content/news/XXXX-XX-XX-mixxx-2_5_3-released.md

```diff
@@ -0,0 +1,61 @@
+Because of the big improvements we have chosen to backport [^1] it to the 2.5 series instead of integrating it in future alfa & beta versions.
+
+Have a nice Mixxx.
+
+[^1]: Back porting is integrating new work into previous releases in order to let those releases profit from the new development.
+
+#### Digital Vinyl System - Timecode System.
+
+A good explanation about Digital Vinyl Systems and Timecode can be found in [this article on the site of Digital DJ Tips](https://www.digitaldjtips.com/a-beginners-guide-to-digital-vinyl-systems/).
+In short summarized: Mixxx plays the music tracks by following the input signal produced by turntables/CD-players (and other Timecode-mediaplayers), so the DJ controls the playback of the digital music by manipulating vinyl or CDs.
+
+In order to create a real vinyl-handling-feeling the produced music needs to follow the inputsignal as close as possible (low latency, fast responsiveness) for eg scratching, spinbacks etc.
+In the past Mixxx tried to follow the input signal as accurately as possible, as there came some updates in Timecodemedia (producing a lot more info), this logic needed to be updated.
+
+Developers have seen the light by choosing the introduction of a [Kalman-filter](https://en.wikipedia.org/wiki/Kalman_filter).
+In short: a Kalman filter can be used to determine the deviation of the actual (observed) point in regard to the pre-calculated point.
+An example: if a train starts at point A at 12:00 to end at point B, 120km further, at a speed of 120 km/h you can calculate all intermediate points and the estimated time of arrival (ETA):
+30km at 12:15, 60km at 12:30, 90km at 12:45 ... Point B at 13:00.
+If the driver uses a GPS, it will start with ETA of 13:00. When the GPS notices that the position of the train at 12:15 is different from the pre-calculated position, eg 28km or 32km,  the GPS knows the train has a higher/lower speed and can recalculate the intermediate points/ETA and display another speed.
+Another example: Kalman filters are also used in weather forecasts to predict the time a storm passes at a certain point (place).
+
+Instead of calculating a couple of points on the trajectories thousands of points can be calculated (and even more), the more pre-calculated and compared points the more accurate the calculation of the deviation [^2] will be.
+
+With the Kalman filter Mixxx can calculate the deviation between the observed point (input from Timecodesignal)  and the pre-calculated point.
+Using this deviation in Mixxx helps Mixxx to properly and more accurately represent eg the rate-slider on the turntable/CD-player.
+
+This is a major improvement useful for everybody, not only for the alpha/beta-users.
+Get all Mixxx-ed up and join our [testing-force](https://mixxx.org/get-involved/).  
```

Comment:
I don't think this summarizes the changes well. 
* #15194 introduced the kalman filter as the new default. There already previously was still a filter ("alpha-beta filter"). The kalman filter is better at producing a stable output while maintaining responsiveness. 
* #15217 Tries to improve sticker drift by trying to account for needle skips.